### ISSUE TITLE
Upgrade from 3.3.3. to 3.3.5 breaks rails migrations

### ISSUE THREAD

**Issue by wallerjake** [capistrano/capistrano#1232]
We are having an issue when upgrading from Capistrano 3.3.3 to 3.3.5. Our Rails migrations no longer run. It seems that the after 'deploy:updated', 'deploy:migrate' is never getting called. Did something change between 3.3.3 to 3.3.5 that could have broken then. I know when we went to 3.3.3 we had to remove the role from our configuration. Here is a sample of our configuration.

```
server 'ip_address', user: 'capistrano', roles: %w{app web basic_work_q_tender}
server 'ip_address', user: 'capistrano', roles: %w{db}, primary: true
server 'ip_address_2', user: 'capistrano', roles: %w{delayed_job search}
```

Please let me know if you need additional information in debugging this. I use Capistrano in another project and the upgrade worked fine and migration still run. So I am thinking it has to be something with this specific configuration.

**Comment by ayastreb** [comment]
Hi!
Could it have something to do with the change in PR #1223? With the fix there I reverted previous fix from PR #1157, and it was included in release 3.3.5

I have already submitted new PR #1315 which should fix both issues.

**Comment by mattbrictson** [comment]
I am pretty sure this has been fixed thanks to #1315. Closing.
